### ISSUE TITLE
fail extra loading check silently

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#3072]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
try {
	execSync(cmd, {stdio: 'inherit'})
}  catch (error) {
    console.log(`Status Code: ${error.status} with '${error.message}'`);
 }
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN && npm run types:publish `);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.59.0](https://github.com/open-wa/wa-automate-nodejs/compare/4.58.0...4.59.0) (2023-03-23)

- Release 4.59.0 [`275bb69`](https://github.com/open-wa/wa-automate-nodejs/commit/275bb69bca7ea586e96be5dbfb08d17f9a4626ff)
- 🐛bug: fix accented filenames breaking file send #3071 [`2973a77`](https://github.com/open-wa/wa-automate-nodejs/commit/2973a77a89fe79dc72fb08672989286f66c484ac)
- ✨ feat: `onGroupApprovalRequest` #3070 [`5c7f4e0`](https://github.com/open-wa/wa-automate-nodejs/commit/5c7f4e08262c5aed11a0780bc1a666163a0b9266)
- ✨ feat: `getGroupApprovalRequests` #3070 [`785a51f`](https://github.com/open-wa/wa-automate-nodejs/commit/785a51fbcbe1bea8f5a385c4853e0e28065ce39f)
- ✨ feat: `setGroupApprovalMode` #3070 [`dabc15b`](https://github.com/open-wa/wa-automate-nodejs/commit/dabc15b6065f754052f87bb306fd6b8f05f07a4f)
- ✨ feat: `rejectGroupJoinRequest` #3070 [`480e1dc`](https://github.com/open-wa/wa-automate-nodejs/commit/480e1dc4a7ba3e7d1638f91d866198c83b5bfa77)
- ✨ feat: `approveGroupJoinRequest` #3070 [`f7d039a`](https://github.com/open-wa/wa-automate-nodejs/commit/f7d039a684d0705f7a262496c9846fa9015bb1f6)
- 👔 biz logic: wait for internal sync to complete before registering listeners [`2f43c08`](https://github.com/open-wa/wa-automate-nodejs/commit/2f43c0819fb2e811d20f7b1ec9306f3a8af6ff61)
- 🏷️ type: `Chat.pic` to get chat icon #3069 [`2329632`](https://github.com/open-wa/wa-automate-nodejs/commit/23296323cedaeb616494f9b3178da70c5862297d)
- 🔧 config: block dit as a crashlog [`27687ee`](https://github.com/open-wa/wa-automate-nodejs/commit/27687ee4683aa58e5b3fb622da3af547c1ec21fe)
- 🥚 patches update [`b54a2de`](https://github.com/open-wa/wa-automate-nodejs/commit/b54a2de6b7e00083ccf3bd8f439e816b351ffab6)
- 🥚 patches update [`42ac5f8`](https://github.com/open-wa/wa-automate-nodejs/commit/42ac5f8ab6553194cdc613e1a8d7d27bb8a83f42)
- 🥚 patches update [`da74535`](https://github.com/open-wa/wa-automate-nodejs/commit/da74535eacd779a7730f3fd49fa93a4e390465e0)
- 🥚 patches update [`df9789d`](https://github.com/open-wa/wa-automate-nodejs/commit/df9789d4d375b86ce768b58c1496df7769615b97)
- 🥚 patches update [`8700fc0`](https://github.com/open-wa/wa-automate-nodejs/commit/8700fc0ad40f3fc8718ad552031836e7cadb2e5d)
- 🥚 patches update [`466f9a9`](https://github.com/open-wa/wa-automate-nodejs/commit/466f9a92db866f6bdd59f17045f386480050ec36)
- updated types-only package [`aab334d`](https://github.com/open-wa/wa-automate-nodejs/commit/aab334db956a6703f23d492bf136367f4bb2c3ea)
- 💚 fix CI [`8dea0fa`](https://github.com/open-wa/wa-automate-nodejs/commit/8dea0fab973a607b5cd87995b422cdf1cf726bff)
